src/frequenz/channels/__init__.py

```diff
@@ -26,11 +26,10 @@
 
 Utilities to work with channels:
 
-* [Merge][frequenz.channels.Merge] and [MergeNamed][frequenz.channels.MergeNamed]:
-  [Receivers][frequenz.channels.Receiver] that merge messages coming from multiple
-  receivers into a single stream.
+* [merge][frequenz.channels.merge]: Merges messages coming from multiple receivers into
+  a single stream.
```

Comment:
To be consistent with `select` documentation
```suggestion
* [merge][frequenz.channels.merge]: Merge messages coming from multiple receivers into
  a single stream.
```